File: src/main/java/com/aurionpro/payrollsystem/dto/organizationSideTicketFunctioning/OrganizationAdminTicketFilterDto.java
package com.aurionpro.payrollsystem.dto.organizationSideTicketFunctioning;

import com.aurionpro.payrollsystem.entity.ticket.TicketStatus;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class OrganizationAdminTicketFilterDto {

	private Long employeeId;
	private Long ticketId;
	private TicketStatus status;
	private String createdAfter;
	private String createdBefore;

}

File: src/app/organization/dto/organization-admin-ticket-filter-dto.ts
import { TicketStatus } from "./ticket-status"; 

export interface OrganizationAdminTicketFilterDto {
  employeeId?: number;
  ticketId?: number;
  status?: TicketStatus;
  createdAfter?: string; // ISO string
  createdBefore?: string; // ISO string
}
